drivers/SmartThings/matter-lock/fingerprints.yml

```diff
@@ -33,3 +33,4 @@ matterGeneric:
     deviceTypes:
       - id: 0x000A # Door Lock
     deviceProfileName: base-lock
+    
```

Comment:
nit: please remove the trailing whitespace here- thank you!

Reply:
Sorry @HunsupJung I know it's a small thing, but I think there is still some whitespace here. You can see if there is or is not based on whether the red circle with a minus shows up at the bottom of the file.

Reply:
Oh, I'm sorry. I thought it was modified, but I didn't merge properly. I will modify it carefully again.

Reply:
I checked here and there's no whitespace. could you check it again by cherry-pick?

Reply:
all good, thanks! Sorry about that